modules/library/src/test/scala/zio/elasticsearch/utils/package.scala

```diff
@@ -18,10 +18,21 @@ package zio.elasticsearch
 
 import zio.json.DecoderOps
 import zio.json.ast.Json
+import zio.schema.{DeriveSchema, Schema}
 
 package object utils {
 
   final implicit class RichString(private val text: String) extends AnyVal {
     def toJson: Json = text.fromJson[Json].toOption.get
   }
+
+  final case class UserDocument(id: String, name: String, address: String, balance: Double, age: Int)
+
+  object UserDocument {
+
+    implicit val schema: Schema.CaseClass5[String, String, String, Double, Int, UserDocument] =
```

Comment:
Do we need `Schema.CaseClass5[...]`?

Reply:
Yes, maybe @mvelimir can provide some info about that?

Reply:
We can't annotate it as `Schema[UserDocument`, but you could remove the type and make sure to put ```// scalafix:ok``` on that line.